.vscode/launch.json

```diff
@@ -13,7 +21,6 @@
             "env": {
                 "PRAVEGA_SENSOR_COLLECTOR_PROPERTIES_FILE": "pravega-sensor-collector/src/test/resources/LeapTest.properties"
             }
-
         }
     ]
 }
```

Comment:
All files should have newlines at the end.

Reply:
Done